lib/rules/one-var.js

```diff
@@ -234,7 +256,7 @@ module.exports = {
 
 		/**
 		 * Determines the current scope (function or block)
-		 * @param {string} statementType node.kind, one of: "var", "let", or "const"
+		 * @param {string} statementType node.kind, one of: "var", "let", "const", "using", or "await using"
```

Comment:
```suggestion
		 * @param {string} statementType one of: "var", "let", "const", "using", or "awaitUsing"
```

Also, the same should be updated in JSDoc of `hasOnlyOneStatement` (line 300).